astropy/table/tests/test_column.py

```diff
@@ -648,8 +648,8 @@ def test_col_unicode_sandwich_create_from_str(Column):
 @pytest.mark.parametrize('Column', (table.Column, table.MaskedColumn))
 def test_col_unicode_sandwich_bytes(Column):
     """
-    Create a bytestring Column from bytes and ensure that it works in Python 3 in
```

Comment:
Please leave the original comment here and below, they provide useful context on what is going on here.